_maps/gaxstation.dm

```diff
@@ -0,0 +1 @@
+#define FORCE_MAP "_maps/gaxstation.json"
```

Comment:
Don't forget to remove this :p

Reply:
![image](https://user-images.githubusercontent.com/5091394/159667044-18db0ebf-ee87-4516-ac5b-e3abadae2672.png)

every map has a .dm hook to the json file :)